BrowserKit/Sources/ToolbarKit/AddressToolbar/BrowserAddressToolbar.swift

```diff
@@ -94,19 +95,19 @@ public class BrowserAddressToolbar: UIView,
         self.toolbarDelegate = toolbarDelegate
         self.isUnifiedSearchEnabled = isUnifiedSearchEnabled
         self.previousLocationViewState = state.locationViewState
-        configure(state: state, isUnifiedSearchEnabled: isUnifiedSearchEnabled)
         updateSpacing(leading: leadingSpace, trailing: trailingSpace)
+        if state.locationViewState.isEditing {
```

Comment:
Would it be possible to set this from `AddressToolbarContainer` using the `calculateToolbarSpace` method? Then we would set the spacing only from one place.